components/common/Link.js

```diff
@@ -33,7 +34,7 @@ const Link = ({
       {React.createElement(
         tag,
         {
-          className: linkCn({ className, disabled, dark, active }),
+          className: linkCn({ className, disabled, dark, active, disableStyles }),
```

Comment:
why not here?
```
className: disableStyles ? className : linkCn({})
```

Reply:
and maybe better naming is `noStyles`

Reply:
because `disableStyles` does not really remove all styles - it puts a special `wir-link--no-styles` class which overwrites some browser defaults.
in other words, there is some logic behind so it's better be extracted.

will rename though